src/compiler/transformers/classFields.ts

```diff
@@ -677,15 +775,19 @@ namespace ts {
                         pendingStatements.push(factory.createExpressionStatement(factory.inlineExpressions(pendingExpressions)));
                     }
 
-                    if (pendingStatements && some(staticProperties)) {
-                        addPropertyStatements(pendingStatements, staticProperties, factory.getInternalName(node));
+                    if (pendingStatements) {

```

Comment:
nit: I don't think the change here is necessary (going from a single conditional to nested conditionals).

Reply:
Made more sense in a previous version 😅